client/components/search/index.jsx

```diff
@@ -38,7 +38,8 @@ module.exports = React.createClass( {
 		onKeyDown: React.PropTypes.func,
 		disableAutocorrect: React.PropTypes.bool,
 		onBlur: React.PropTypes.func,
-		searching: React.PropTypes.bool
+		searching: React.PropTypes.bool,
+		forceCloseButton: React.PropTypes.bool
```

Comment:
As an example of how search should behave with the current set of props look at `/sites` (which has search opened by default), `/design`, or the site-picker. I feel we should be consistent here and avoid complicating `<Search>` even more.